ergo/distributions/conditions.py

```diff
@@ -74,6 +94,26 @@ def describe_fit(self, dist):
         description["p_in_interval"] = float(self.actual_p(dist))
         return description
 
+    def get_normalized(self, true_min: float, true_max: float):
```

Comment:
`scale_min`, `scale_max`, and `scale_range` would be more intuitive to me here. `self.min - true_min` was difficult to follow for me.